client/dashboard/components/connect/test/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
```

Comment:
Hooray tests!

AFAIK we've made the decision to write new tests using `testing-library/react`, not Enzyme. Gutenberg has made this move, as well as other teams at a8c - though I can't find anything "official" on our P2.

This is a rather small test, so I'll consider the rewrite optional for landing this PR. I'll create a follow up issue to migrate our Enzyme tests as well.

Reply:
Thanks for creating an issue to migrate the enzyme tests Jeff - seems like a great cooldown task.

Reply:
I've changed this test to use `testing-library/react` (07a74620) - I like it :-) however I didn't migrate the other test because I don't want to migrate the entire suite in this PR.